### PR TITLE
Feature/point intercept tests

### PR DESCRIPTION
**Pull request by elodszopos**
Added unit tests for the player side! It checks if point interaction works as it should. I'm now also making use of the newly released test-utils (thank you @edeustace!)

### CODE REVIEW

**Comment by edeustace** [comment]
few test setup changes, moved `@pie-lib/test-utils` to the repo root, we only want `enzyme` to be installed at the root level so that it gets configured correctly.

File: packages/point-intercept/jest.config.js [hunk 1]
```diff
@@ -0,0 +1,5 @@
+module.exports = {
```
Comment (edeustace) [inline packages/point-intercept/jest.config.js]: all  test related stuff can go - it's all configured at the mono-repo root.

File: packages/point-intercept/src/__tests__/main.test.js [hunk 2]
```diff
@@ -0,0 +1,142 @@
+import * as React from 'react';
+import { PlotPoints } from '@pie-lib/charting';
+import { shallowChild } from '@pie-lib/test-utils';
+import Controls from '../controls';
+import CorrectAnswerToggle from '@pie-lib/correct-answer-toggle';
+import { Feedback } from '@pie-lib/render-ui';
+import Main from '../main';
+
+describe('Main', () => {
```
Comment (edeustace) [inline packages/point-intercept/src/__tests__/main.test.js]: there'd be a couple of other functions we could test? `deleteSelection`, `movePoint`?

Comment (elodszopos) [reply packages/point-intercept/src/__tests__/main.test.js]: They are tested already. 
You can find the relevant tests -

here: https://github.com/pie-framework/pie-ui/pull/14/files#diff-88f2e0a2357f6a20ba964c8e946d5d1dR85
and here: https://github.com/pie-framework/pie-ui/pull/14/files#diff-88f2e0a2357f6a20ba964c8e946d5d1dR108

I think I got pretty good coverage here for the internal functionality.

File: packages/point-intercept/jest.config.js (hunk 1 above)
Comment (elodszopos) [reply packages/point-intercept/jest.config.js]: Taken care of!

**Pull request merged**